.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -406,6 +406,7 @@ multi-asic-t1-lag:
   - snmp/test_snmp_loopback.py
   - snmp/test_snmp_pfc_counters.py
   - snmp/test_snmp_queue.py
+  - snmp/test_snmp_queue_counters.py
```

Comment:
This is currently getting skipped in the PR checker for multi-asic VS

Reply:
seems like this is broken for multi-aisc and is being skipped. https://github.com/sonic-net/sonic-mgmt/issues/14007

Reply:
> seems like this is broken for multi-aisc and is being skipped. #14007

Yes, we got the same error as mentioned in #14007 once this test case was added to T2. This PR will fix this issue.